### PR TITLE
entrega 4 wnicolait

### PR DESCRIPTION
**Pull request by wnicolait**

### CODE REVIEW

File: Assignment4/wnicolait-80097/task06.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.1: Create a new class named "University"**"""
+
+nsP = Namespace("http://Universithy/")
+nsPer= Namespace("http://Person/")
+g.add((nsP.Universithy, RDF.type, RDFS.Class))
```
Comment (dachafra) [inline Assignment4/wnicolait-80097/task06.py:37]: University, not Universithy.

File: Assignment4/wnicolait-80097/task06.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+"""Create a new class named Researcher"""
+
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.1: Create a new class named "University"**"""
+
+nsP = Namespace("http://Universithy/")
```
Comment (dachafra) [inline Assignment4/wnicolait-80097/task06.py:35]: these namespaces are not right

File: Assignment4/wnicolait-80097/task06.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+nsPer= Namespace("http://Person/")
+g.add((nsP.Universithy, RDF.type, RDFS.Class))
+g.add((nsPer.Person, RDF.type, RDFS.Class))
```
Comment (dachafra) [inline Assignment4/wnicolait-80097/task06.py:38]: why are you creating the class person?

File: Assignment4/wnicolait-80097/task06.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+for s, p, o in g:
+  print(s,p,o)
+
+
+
+"""**TASK 6.2: Add "Researcher" as a subclass of "Person"**"""
+
+
+
+app = nsPer.Researcher
```
Comment (dachafra) [inline Assignment4/wnicolait-80097/task06.py:48]: no no, review it!

File: Assignment4/wnicolait-80097/task06.py [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+     says= nsPer.says
+     inv=Literal("Jane Smith")
+     print(app)
+
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
```
Comment (dachafra) [inline Assignment4/wnicolait-80097/task06.py:53]: ufff read carefully the tasks please, these lines are copy-paste from previous tasks

File: Assignment4/wnicolait-80097/task06.py [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+triple =(app,says,inv)
+g.add(triple)
+search = g.triples((None,says,None))
+for subj,pred,obj in search:
+  print(subj,pred,obj)
+
+"""**TASK 6.4: Add to the individual JaneSmith the fullName, given and family names**"""
```
Comment (dachafra) [inline Assignment4/wnicolait-80097/task06.py:61]: what about the other properties? given and family name

File: Assignment4/wnicolait-80097/task06.py [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+print(resource)
+g.add(resource)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+app = nsP.Upm
+says= nsPer.says
+inv=Literal("John Smith")
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+resource = (app, VCARD.FN, inv)
```
Comment (dachafra) [inline Assignment4/wnicolait-80097/task06.py:77]: you are saying here: UNiversity has a full name that is John Smith

**Comment by dachafra** [comment]
Please, review my comments, and do the tasks from Scratch after reading and testing carefully the examples. The lines you program do not have any sense

**Review by dachafra — CHANGES_REQUESTED** [review]
review from scratch

File: Assignment4/wnicolait-80097/task06(1).py [hunk 8]
```diff
@@ -32,51 +30,42 @@
...
-g.add((nsPer.Person, RDF.type, RDFS.Class))
+g.add((ns.University, RDF.type, RDFS.Class))
 for s, p, o in g:
   print(s,p,o)
 
 
 
 """**TASK 6.2: Add "Researcher" as a subclass of "Person"**"""
 
+g.add((ns.Person, RDF.type, RDFS.subClassOf))
```
Comment (dachafra) [inline Assignment4/wnicolait-80097/task06(1).py:41]: you are saying here that person is type of sublcassof, does not make sense

File: Assignment4/wnicolait-80097/task07(1).py [hunk 9]
```diff
@@ -37,42 +36,37 @@
   initNs = { "clave": VCARD}
 )
 
-for r in g.query(query2):
+for r in g.query(q):
   print(r)
 
 """**TASK 7.2: List all individuals of "Person" with RDFLib and SPARQL (remember the subClasses)**"""
```
Comment (dachafra) [inline Assignment4/wnicolait-80097/task07(1).py:42]: where is the sparql query?

File: Assignment4/wnicolait-80097/task07(1).py [hunk 10]
```diff
@@ -37,42 +36,37 @@
...
 """**TASK 7.3: List all individuals of "Person" and all their properties including their class with RDFLib and SPARQL**"""
 
 from rdflib.plugins.sparql import prepareQuery
 
 
-VCARD = Namespace("http://somewhere#/Person")
+VCARD = Namespace("http://somewhere#/")
 
 
 qt = prepareQuery('''
```
Comment (dachafra) [inline Assignment4/wnicolait-80097/task07(1).py:60]: this does not make sense, please read carefully what the task7.3 is asking for

**Review by dachafra — CHANGES_REQUESTED** [review]
make the corresponding changes, there are many issues to solve

**Comment by ocorcho** [comment]
Given that the changes requested by @dachafra 6 days ago have not been dealt with, we are closing this pull request.
If you want this to be considered further, please contact us by e-mail.

**Pull request closed without merging**